sisl/io/siesta/out.py

```diff
@@ -249,18 +249,39 @@ def read_force(self, last=True, all=False):
         all: bool, optional
            return a list of all forces (like an MD)
            If `True` `last` is ignored
+        total: bool, optional
+            return the total forces instead of the atomic forces.
+        maxF: bool, optional
+            whether only the maximum atomic force should be returned for each step.
+            If `True`, `total` is ignored.
 
+            Setting it to `True` is equivalent to `max(outSile.read_force())` in case atomic forces
+            are written in the output file (`WriteForces .true.` in the fdf file)
+
+            Note that this is not the same as doing `max(outSile.read_force(total=True))` since
+            the forces returned in that case are averages on each axis.
+
+            
         Returns
         -------
         numpy.ndarray or None
             returns ``None`` if the forces are not found in the
             output, otherwise forces will be returned
+
+            The shape of the array will be different depending on the type of forces requested:
+                - atomic (default): (nMDsteps, nAtoms, 3)
+                - total: (nMDsteps, 3)
+                - maxF: (nMDsteps, )
```

Comment:
If both are queried then it should probably be `(nMDsteps, 4)`, or what do you think?

Reply:
I don't know, do you think there will be a case in which it will be needed to do the same operation on both at the same time?

Otherwise we could just return [ total, maxF] (or the opposite). The user could zip them in a loop and still kind of have them separately. I mean

```python
for totalF, max in zip(total, maxF):
     # totalF[0] to get Fx, etc...
```
vs

```python
for Fx, Fy, Fz, max in forces:
```

I don't know what's best. Whatever you prefer.

Reply:
I just realized that you can go like 

```python
for *totalF, max in forces:
```
so my point is useless (unless you consider that begginers in python will not come up with this probably). But I think it's more a matter of the two things not quite belonging to the same property that confuses me if you put them together in an array. Idk

Reply:
yeah, probably return a tuple, yeah do that :)